services/movie_session.py

```diff
@@ -0,0 +1,54 @@
+from django.db.models import QuerySet
+from db.models import MovieSession
+from typing import Any
+
+
+def create_movie_session(
+        movie_show_time: Any,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> None:

```

Comment:
The parameters `movie_show_time`, `movie_id`, and `cinema_hall_id` have default values of `None`. This could lead to issues if the function is called without arguments. Consider adding validation to ensure these parameters are provided and are valid. Additionally, `movie_show_time` should have a more specific type, such as `datetime`, instead of `Any`.